Fixes #10846 TypeError: Cannot read property 'substring' of undefined

xgin opened the pull request:
Signed-off-by: Eugene Kazakov <ekazakov@plesk.com>

### Description

Fixes #10846
The issue could be reproduced with phpMyAdmin versions 4.8.5 and 5.0.0-dev with the following steps:

1. Create a database and a table (empty is okay)
2. Click on the database name
3. Refresh the browser (notice the icon "Open new phpMyAdmin window" in the right bottom corner is present)
4. Click on the table name (notice the icon "Open new phpMyAdmin window" disappeared)
5. Click "Operations"
6. "Rename table to" enter a new name and click "Go"

**Javascript TypeError: Cannot read property 'substring' of undefined**
```json
{
    "pma_version": "4.8.5",
    "browser_name": "CHROME",
    "browser_version": "74.0.3729.131",
    "user_os": "Mac",
    "server_software": "nginx/1.14.1",
    "user_agent_string": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36",
    "locale": "en",
    "configuration_storage": "enabled",
    "php_version": "7.3.5",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "Cannot read property 'substring' of undefined",
        "stack": [
            {
                "func": "Object.refreshMain",
                "line": "151",
                "column": "23",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            },
            {
                "func": "Object.success",
                "line": "114",
                "column": "39",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            },
            {
                "func": "u",
                "line": "2",
                "column": "27457",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            },
            {
                "func": "Object.fireWith [as resolveWith]",
                "line": "2",
                "column": "28202",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            },
            {
                "func": "k",
                "line": "2",
                "column": "77651",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            },
            {
                "func": "XMLHttpRequest.",
                "line": "2",
                "column": "79907",
                "uri": "index.php?v=4.8.5",
                "scriptname": "index.php"
            }
        ],
        "uri": "index.php?"
    },
    "script_name": "index.php",
    "microhistory": null
}
```

The root cause of the issue is a corrupted layout: missing closing `div` - because of it, `selflink` appears inside `page_content` and then is replaced with the content of the next page.
I would like to also make the redirect function more reliable and use a fallback with the current location in case the link is lost again.

Let me know if it makes sense to submit the PR to a released version branch.

williamdes commented:
Hi @xgin, thanks for the contribution
Good catch !
Can you please rebase the branch and change the target of the PR to `QA_4_8` ?

I think there is another occurrence in the file, can the `table_header` twig file be edited instead ?

xgin committed:
Bug #10846 TypeError: Cannot read property 'substring' of undefined

Signed-off-by: Eugene Kazakov <ekazakov@plesk.com>

xgin commented:
Hi @williamdes, thank you for the feedback!
I rebased the branch and changed the fix: now `table_header.twig` includes the full layout with closing tags.

williamdes reviewed (approved):
LGTM
Great work :100:!

williamdes commented:
Merged, thanks for the contribution @xgin !
